specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/logic.json

```diff
@@ -2486,20 +2743,20 @@
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/IntegrationAccountAgreement"
+              "$ref": "#/definitions/BatchConfiguration"
             }
           }
         }
       },
       "put": {
         "tags": [
-          "IntegrationAccountAgreements"
+          "IntegrationAccountBatchConfigurations"
         ],
-        "operationId": "Agreements_CreateOrUpdate",
-        "description": "Creates or updates an integration account agreement.",
+        "operationId": "IntegrationAccountBatchConfigurations_CreateOrUpdate",
```

Comment:
Often a createOrUpdate that returns a 201, or a delete that returns a 204 - may be a indicative of a long running operation (x-ms-long-running-operation). I see none of the other create/delete functions use a long running API - but just to be sure, can you confirm that that's the case here?